### ISSUE TITLE
Add info about "gfx" folder to Readme

### ISSUE THREAD

**Issue by as3boyan** [HaxeFlixel/flixel-ui#88]
Not a big deal actually

I was digging RPG Interface demo code
and seen that specified paths in XML files are relative to `assets/gfx`
So I was looking for info about this folder in Readme
https://github.com/HaxeFlixel/flixel-ui#quick-project-setup
and
then found it
https://github.com/HaxeFlixel/flixel-ui#xml-layout-basics

And I thought maybe you guys can add a note on `gfx` to
https://github.com/HaxeFlixel/flixel-ui#quick-project-setup

`In your openfl assets folder, create an "xml" directory`
Maybe add something like "create a gfx folder"

**Comment by larsiusprime** [comment]
Yeah, it could be made more clear. And in any case it should be easy for me to make these user-configurable, so you could just do:

FlxUI.xmlFolder = "path/to/xml";
FlxUI.assetFolder = "path/to/gfx";

And then it recognizes that from then on.

**Comment by vantreeseba** [comment]
@larsiusprime This is already kind of customizable in your project.xml.

```
<assets path="assets/ui" rename="assets/xml" include="*.xml"/>
```

**Comment by J5lx** [comment]
@vantreeseba That's actually quite a neat trick. I'd still prefer @larsiusprime's solution though since it's less hacky.

**Comment by Tiago-Ling** [comment]
Why not use HaxeFlixel's default asset folders, that is `data` (in place of `xml`) and `images` (in place of `gfx`)?

This would make more sense since HaxeFlixel creates new projects in with this asset structure.

**Comment by larsiusprime** [comment]
Yeah not a bad idea. Next time I get a chance to clear out some flixel-ui
issues I'll address this.

On Wed, Apr 1, 2015 at 5:28 AM, Tiago Ling Alexandre <
notifications@github.com> wrote:

> Why not use HaxeFlixel's default asset folders, that is data (in place of
> xml) and images (in place of gfx)?
>
> This would make more sense since HaxeFlixel creates new projects in with
> this asset structure.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HaxeFlixel/flixel-ui/issues/88#issuecomment-88431443>.
>



-- 
www.fortressofdoors.com -- Games, Art, Design
